core/flamingock-core-api/src/main/java/io/flamingock/api/annotations/EnableFlamingock.java

```diff
@@ -213,8 +213,8 @@
     /**
      * If true, the annotation processor will validate that all code-based changes
      * (classes annotated with @Change) are mapped to some stage. When unmapped changes
-     * are found and this flag is true, a RuntimeException is thrown. Default is false
-     * (only a warning is emitted).
+     * are found and this flag is true, a RuntimeException is thrown at compilation time. Default is true.
+     * When false, only a warning is emitted.
```

Comment:
If true, the annotation processor will validate that all code-based changes
classes annotated with @Change) are mapped to some stage. When unmapped changes
are found and this flag is true**(default)**, a RuntimeException is thrown at compilation time. 
When false, only a warning is emitted.


Just small polish to make it more natural :)